docs/components/home/index.tsx

```diff
@@ -52,67 +61,95 @@ const components = {
-                    <ProsOrConsList items={cons} label={consLabel} type="con" />
-                  </div>
-                </div>
-              ))}
+                  ),
+                )}
+              </div>
+
+              <div className="mt-4 mb-8 flex justify-center">
+                <Button
+                  as="a"
+                  href="/docs/getting-started"
+                  className="mt-8 sm:mt-10"
+                  role="button"
+                >
+                  Get started
+                </Button>
+              </div>
             </div>
 
-            <div className="mt-4 mb-8 flex justify-center">
-              <a href="/docs/getting-started" title="" className="inline-flex px-8 py-4 mt-8 text-lg font-bold transition-all duration-200 bg-quint-purple text-black rounded sm:mt-10 font-pj hover:bg-[#2d0075] hover:text-quint-purple focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-900" role="button">
-                Get started
-              </a>
+            <div className="xl:col-span-1 text-lg">
+              <CodeSample components={components} />
+              <div className="flex justify-center">
+                <Button
+                  as="button"
+                  className={isViolationVisible ? 'hidden' : 'visible'}
+                  variant="secondary"
+                  onClick={handleToggleVisibility}
+                >
+                  Find bug
+                </Button>
+              </div>
+              <div
+                className={`transition-all duration-1000 ease-out overflow-hidden ${
+                  isViolationVisible
+                    ? 'opacity-100 max-h-screen'
+                    : 'opacity-0 max-h-0'
+                }`}
+              >
+                <ViolationSample components={components} />
+              </div>
             </div>
           </div>
 
-          <div className="xl:col-span-1 text-lg">
-            <CodeSample components={components} />
-            <div className="flex justify-center">
-              <button onClick={handleToggleVisibility} className={`transition-all duration-500 ease-out overflow-hidden ${isViolationVisible ? 'hidden' : 'visible'} inline-flex px-4 py-2 font-bold transition-all duration-200 bg-quint-purple text-black rounded font-pj hover:bg-[#2d0075] hover:text-quint-purple focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-900`}>
-                Find bug
-              </button>
-            </div>
-            <div className={`transition-all duration-1000 ease-out overflow-hidden ${isViolationVisible ? 'opacity-100 max-h-screen' : 'opacity-0 max-h-0'}`} >
-              <ViolationSample components={components} />
-            </div>
-          </div>
-        </div>
+          <NewsletterSignupBanner />
 
-        <div className="mt-8 mb-8 flex justify-center">
-          {informalSystemsLogo()}
+          <div className="mt-8 mb-8 flex justify-center">
+            {informalSystemsLogo()}
```

Comment:
I think the logo looks a bit weird after the new banner. Perhaps we can move it to the footer?